queries.sql

```diff
@@ -8,3 +8,39 @@ select name, escape_attempts from animals where weight_kg>10.5
 select * from animals where neutered is true
 select * from animals where name != 'Gabumon'
 select * from animals where weight_kg between 10.4  and 17.3
+
```

Comment:
- [ ] @JakariaJishan kindly add a query to change the species column to unspecified and rollback the change, while at it kindly don't forget to add a query to verify the change was done before rollback and a query to verify the change was undone
![image](https://user-images.githubusercontent.com/61408860/229921413-ac9ba689-c7e0-4884-a639-8fac24935177.png)